Libs/Mesh/Mesh.cpp

```diff
@@ -677,7 +677,7 @@ std::vector<std::string> Mesh::getFieldNames() const
   return fields;
 }
 
-Mesh& Mesh::setField(std::string name, Array array)
+Mesh& Mesh::setField(std::string name, Array array, bool multi)
```

Comment:
What is done with the multi parameter?

Reply:
There used to be a warning if your tuples were more than 1. This was an optional parameter to suppress this warning. I've removed it though.